tests/v2/validation/rbac/rbac_test.go

```diff
@@ -345,6 +406,7 @@ func (rb *RBTestSuite) TestRBAC() {
 		{"Cluster Member", roleMember, standardUser},
 		{"Project Owner", roleProjectOwner, standardUser},
 		{"Project Member", roleProjectMember, standardUser},
+		{"Project Manage Member", roleManageProjectMember, standardUser},
```

Comment:
Since, we have the new tests in `TestRBACAdditional` suite, why are we adding this here? This will run the TestRBAC for all the tests cases with Project Manage Member role..